sessionctx/variable/session.go

```diff
@@ -1000,11 +1000,9 @@ type SessionVars struct {
 	// EnablePaging indicates whether enable paging in coprocessor requests.
 	EnablePaging bool
 
-	// StmtStats is used to count various indicators of each SQL in this session
-	// at each point in time. These data will be periodically taken away by the
-	// background goroutine. The background goroutine will continue to aggregate
-	// all the local data in each session, and finally report them to the remote
-	// regularly.
+	// StmtStats is a pointer to session.stmtStats, and the life cycle of StmtStats
+	// is controlled by session.
+	// See session.stmtStats for more details.
 	StmtStats *stmtstats.StatementStats
```

Comment:
How about remove this and add a new method in `sessionctx.Context`?

Reply:
In fact we only need to maintain `StatementStats` in `tidb/session`, other `sessionctx.Context` implementations (such as `TiDBContext`, `session in br` etc...) do not need to add this concept.
If we modify the interface, these implementations need to add corresponding method, which seems unnecessary...

Reply:
We can make this a common concept. Not only the Top SQL but other parts like Slow Query and Statement will rely on these data one day.

Reply:
done